mw-webapp/src/service/UserService.ts

```diff
@@ -22,6 +26,8 @@ import {logToConsole} from "src/utils/logToConsole";
 import {PartialWithUuid} from "src/utils/PartialWithUuid";
 
 export const PATH_TO_USERS_COLLECTION = "users";
+const USERS_LIMIT = 10;
```

Comment:
why users limited?

Reply:
Because we using limit for pagination

Reply:
naming

Reply:
maybe it is related to pagination and it does not related to any limit